code/game/machinery/cloning.dm

```diff
@@ -433,8 +433,7 @@
 	if(H.mind in SSticker.mode.syndicates)
 		SSticker.mode.update_synd_icons_added()
 	if(H.mind in SSticker.mode.cult)
-		SSticker.mode.add_cultist(occupant.mind)
-		SSticker.mode.update_cult_icons_added() //So the icon actually appears
+		SSticker.mode.update_cult_icons_added(H.mind) //So the icon actually appears
 		SSticker.mode.update_cult_comms_added(H.mind) //So the comms actually appears

```

Comment:
I assume `add_cultist` was removed because it was redundant?

Reply:
Yeah if the mind is in SSticker.mode.cult then it's already a cultist.